paypal_account.go

```diff
@@ -53,6 +54,11 @@ func (paypalAccount *PayPalAccount) GetImageURL() string {
 	return paypalAccount.ImageURL
 }
 
+// GetRevokedAt returns the revoked at time
+func (paypalAccount *PayPalAccount) GetRevokedAt() *time.Time {
```

Comment:
Any reason why we would need a getter?

Reply:
I was doing this to expose revokedAt in the service verify in PS. Something like this 
`mtd, err := b.verifier.AdapterVerify(ctx, vBody)
	if err != nil {
		return “”, err
	}
	if ppAct, ok := mtd.(*braintree.PayPalAccount); ok {
		if ppAct.GetRevokedAt() != nil {
			return “”, errors.New(“Token is invalid, it has been revoked.“)
		}
	}
	return mtd.GetToken(), nil`
	 But I guess I can simply do this by accessing revokedAt field without having to use a getter.

Reply:
I would also prefer accessing the field directly as this getter does not provide any value here